mmv1/products/networksecurity/SecurityProfileGroup.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'SecurityProfileGroup'
+base_url: '{{parent}}/locations/{{location}}securityProfileGroups'
+create_url: '{{parent}}/locations/{{location}}/securityProfileGroups?securityProfileGroupId={{name}}'
+self_link: '{{parent}}/locations/{{location}}/securityProfileGroups/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  A security profile group defines a container for security profiles.
+references:
+  !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/firewall/docs/reference/network-security/rest/v1beta1/organizations.locations.securityProfileGroups'
+  guides:
+    'Security profile groups overview': 'https://cloud.google.com/firewall/docs/about-security-profile-groups'
+    'Create and manage security profile groups': 'https://cloud.google.com/firewall/docs/configure-security-profile-groups'
+async: !ruby/object:Api::OpAsync
```

Comment:
Most of these fields are ignored by the generator - https://googlecloudplatform.github.io/magic-modules/develop/resource/ shows the minimal set of fields that can be set.

Reply:
@melinath what field are you exactly referring to?

Reply:
Got it now from your other comment. Please, let me know if things look as expected now and in case resolve the comments. Thanks!

Reply:
Answers in my other comment

Reply:
responded on the other PR - TLDR; I wouldn't expect include_project to be necessary on most resources and would prefer for `include_project` to be removed from this block unless you're sure it's required.

Reply:
Yep, I've tried. Unfortunately,  also this resource -as all the other IPS firewall resources- wants the include_project directive. Otherwise, tests fail.